documentation/docs/admin/roles/access-control/labels.md

```diff
@@ -1,41 +1,55 @@
 
+**Examples**
 
-| **Label Type**| **Object**| **Label name **| **Example** |                                                                                                
-|----------|--------|-------|------------------------------|
-| **Standard**  | Node   | node_id |123|                                          
-|          | Service|service_type   |   - mysql, mongodb, postgresql etc.                                     
-| **Custom**| Node, Service, Agent| Any string matching regular expression: <br /> [a-zA-Z_][a-zA-Z0-9_]*. <br /> Also, it cannot start with two underscores.| owner="joe"<br/> _rack="12345"|
+| **Label Type**| **Object**| **Label name ** | **Example** |                                                                                                
+|---------------|-----------|-----------------|--------------------------------------|
+| **Standard**  | Node      | node_id         | 5bdfb1b4-c6c4-4086-83a2-e8daa0b84d4b |                                          
+| **Standard**  | Service   | service_type    | mysql, mongodb, postgresql etc.      |
+| **Custom**    | Node, Service, Agent | Any string matching the regular expression: <br /> [a-zA-Z_][a-zA-Z0-9_]*. <br /> Also, it cannot start with two underscores.| owner="joe"<br/> _rack="12345"|
 
 ## Adding labels
 
-You can add custom or standard labels in PMM while adding a service for monitoring in PMM.
+You can add standard or custom labels while adding a service to monitoring in PMM.
 
 ### Using PMM UI
 
-To set the labels using the user interface:
+To set the labels via the user interface:
 {.power-number}
 
 1. From the **Main** menu, go to **PMM Configuration > PMM Services > Add Service**.
 
-2. Select the service you want to add to PMM for monitoring. The page to add the service opens.
+2. Select the service you want to monitor. The page to add the service opens.
+
+3. Enter the service parameters such as *Hostname, Service name, Port, Username, Password*, etc.
+
+4. Enter standard labels via the input section `Labels`.
 
-3. Enter the details such as *Hostname, Service name, Port, Username, Password,* etc., along with Label or Custom labels.
+5. Enter custom labels via section `Custom labels`.
 
- ![!](../../../images/PMM_access_control_add_labels_services.png)
+![PMM Inventory - Add Service](../../../images/lbac/PMM_access_control_add_labels_services.png)
 
 ### Using pmm-admin
 
- You can also assign labels using [pmm-admin](../../../use/commands/pmm-admin.md).
+You can also add standard and custom labels using [pmm-admin](../../../use/commands/pmm-admin.md).
+
+
+## Modifying labels
+
+Currently, PMM does not support modifying standard labels. To modify a standard label, you first need to remove the service and re-add it with a new set of standard labels.
+
+However, you can modify custom labels of a service via an [API endpoint](https://percona-pmm.readme.io/reference/changeservice).
```

Comment:
that endpoint allows to modify some standard labels like `cluster`, `environment`, `replication_set` and `external_group`

Reply:
it's on UI as well

Reply:
Done.